reference/5.1/Microsoft.PowerShell.Utility/Get-Member.md

```diff
@@ -22,167 +21,232 @@ Get-Member [-InputObject <PSObject>] [[-Name] <String[]>] [-MemberType <PSMember
+StartupType               Property      Microsoft.PowerShell.Commands.ServiceStartupType {get;set;}
 Status                    Property      System.ServiceProcess.ServiceControllerStatus Status {get;}
+UserName                  Property      System.String {get;set;}
+ToString                  ScriptMethod  System.Object ToString();
 ```
 
-This command displays the properties and methods of the process objects (System.ServiceProcess.ServiceController) that are generated by the Get-Service cmdlet.
-
-The command uses the pipeline operator (|) to send the output of a **Get-Service** command to **Get-Member**.
+### Example 2: Get members of service objects
 
-Because the **Get-Member** part of the command does not have any parameters, it uses all of the default values.
-As such, it gets all member types, but it does not get static members and does not display intrinsic members.
+This example gets all of the members (properties and methods) of the service objects retrieved by
+the `Get-Service` cmdlet, including the intrinsic members, such as **PSBase**, **PSObject**, and the
+**get_** and **set_** methods.
 
-### Example 2: Get members of service objects
-```
-PS> Get-Service | Get-Member -Force
-PS> (Get-Service Schedule).PSBase
+```powershell
+Get-Service | Get-Member -Force
+(Get-Service Schedule).PSBase
 ```
 
-This example gets all of the members (properties and methods) of the service objects (System.ServiceProcess.ServiceController) retrieved by the Get-Service cmdlet, including the intrinsic members, such as PSBase and PSObject, and the get_ and set_ methods.
-
-The first command uses the **Get-Service** cmdlet to get objects that represent the services on the system.
-It uses a pipeline operator (|) to pass the service objects to the **Get-Member** cmdlet.
+The `Get-Member` command uses the **Force** parameter to add the intrinsic members and
+compiler-generated members of the objects to the display. You can use these properties and methods
+in the same way that you would use an adapted method of the object. The second command shows how to
+display the value of the PSBase property of the Schedule service.
 
-The **Get-Member** command uses the *Force* parameter to add the intrinsic members and compiler-generated members of the objects to the display.
-**Get-Member** gets these members, but it hides them by default.
+### Example 3: Get extended members of service objects
 
-You can use these properties and methods in the same way that you would use an adapted method of the object.
-The second command shows how to display the value of the PSBase property of the Schedule service.
+This example gets the methods and properties of service objects that were extended by using a
+`Types.ps1xml` file or the `Add-Member` cmdlet.
 
-### Example 3: Get extended members of service objects
+```powershell
+Get-Service| Get-Member -View Extended
 ```
-PS> Get-Service| Get-Member -View Extended
-TypeName: System.ServiceProcess.ServiceController
+
+```Output
```

Comment:
On 5.1, I get two additional values, RequiredServices and ToString.

Reply:
Fixed